deployment/hasura/metadata/databases/tables/merlin/plan_derivation_group.yaml

```diff
@@ -0,0 +1,86 @@
+table:
+  name: plan_derivation_group
+  schema: merlin
+configuration:
+  custom_name: "plan_derivation_group"
+object_relationships:
+- name: plan
+  using:
+    foreign_key_constraint_on: plan_id
+- name: derivation_group
+  using:
+    foreign_key_constraint_on: derivation_group_name
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: '*'
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: [plan_id, derivation_group_name]
+      check: {}
+  - role: user
+    permission:
+      columns: [plan_id, derivation_group_name]
```

Comment:
`acknowledged` column should be exposed on `insert`

Reply:
Actually, thinking about it -- the real reason I made this comment is because should `acknowledged` default to `true` and not `false`? I don't think a user would expect a notification for a derivation group they just added.

Reply:
Yep! This was a mistake I made when I sent you those screenshots but what is in the repository right now is a default to `true` for the `acknowledged` column. Just like we didn't expose `last_acknowledged_at` on insert previously, I would say it is consistent to also not expose `acknowledged` on insert, and instead only on update.

Reply:
Yep, full agree, I had missed that the `default` had changed between screenshot and commit. 👍